### ISSUE TITLE
"ui-test" build-agent label is deprecated, please switch to "centos-7"

### ISSUE THREAD

**Issue by fredg02** [eclipse/Xpect#280]
See line 13: https://github.com/eclipse/Xpect/blob/master/Jenkinsfile#L13

Announcement was posted on cbi-dev and cross-project-issues-dev mailing lists:
https://www.eclipse.org/lists/cbi-dev/msg02266.html

**Comment by cdietrich** [comment]
@fredg02 is there somewhere a step by step guide what to do?

**Comment by fredg02** [comment]
If it has not happened in all Jenkinsfiles on all branches already, replace the label "ui-test" with "centos-7".

**Comment by cdietrich** [comment]
this does not work cause wrong maven and java version are used
how can i use java 8?

**Comment by fredg02** [comment]
```
node('centos-7') {
  jdk = tool 'adoptopenjdk-hotspot-jdk8-latest'
  env.JAVA_HOME = "${jdk}"
  stage {
    sh "${jdk}/bin/java -version"
  }
}
```

**Comment by cdietrich** [comment]
thx.

centos was already used.
java 8/11 fix will be done via #286
